src/sentry/static/sentry/app/views/projectSettings/index.jsx

```diff
@@ -170,11 +171,22 @@ const ProjectSettings = createReactClass({
           </ul>
         </div>
         <div className="col-md-10">
-          {React.cloneElement(this.props.children, {
-            setProjectNavSection: this.props.setProjectNavSection,
-            project,
-            organization: this.context.organization,
-          })}
+          {access.has('project:write') ? (
```

Comment:
are there any settings inside this page that they're allowed to see? the code looks good to me

Reply:
No, pretty sure the whole section is only supposed to be visible for users with write access